app/uk/gov/hmrc/tariffclassificationfrontend/views/confirm_supressed_case.scala.html

```diff
@@ -0,0 +1,38 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+@import uk.gov.hmrc.tariffclassificationfrontend.models.Case
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.includes.main
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials._
+
+@(c: Case)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@main(bodyClasses = None) {
+    @case_heading(c)
+
+    <div class="grid-row">
+        <div class="column-full">
+            <h3 class="heading-large mt-0">This case has been suppressed</h3>
+
+            <div class="heading-medium mt-45">Next Steps</div>
+
+            <p>Go back to <a href="@uk.gov.hmrc.tariffclassificationfrontend.controllers.routes.QueuesController.queue("gateway")">@messages("cases.menu.gateway-cases")</a></p>
```

Comment:
`uk.gov.hmrc.tariffclassificationfrontend.controllers.routes.QueuesController` can be imported

Reply:
Done